### DISCUSSION TITLE
Performance question

### DISCUSSION THREAD

**Discussion by ghost** [Live-Charts/LiveCharts2#541] [Q&A]
I'm using a CartesianChart with nine series. X-axis is datetime. I made some performance tests and found that an update of the graph takes about 800 ms. I measured the time between the UpdateStarted and the UpdateFinished event with DateTime.Now.Ticks. The measured time is independend of the device. I checked it with the android emulator, a Samsung tablet, a Samsung mobile, a Huawei tablet, UWP and iPad simulator. The update time is always around 800 ms. The time is although independend of the number of series.

How can I improve the timeing. My aim is to have an update within 200 ms.

**Comment by beto-rodriguez** [comment]
The high performance package is not ready yet, but the library will provide a solution for that, see #486.
